internal/install/recipe_installer.go

```diff
@@ -178,13 +206,18 @@ func (i *RecipeInstaller) Install() error {
 
 func (i *RecipeInstaller) installRecipesWithPrompts(m *types.DiscoveryManifest, recipes []types.Recipe, entityGUID string) (bool, error) {
 	errorsEncountered := false
+	log.Debugf("Installing recipes with prompts...")
 
 	for _, r := range recipes {
+		log.Debugf(fmt.Sprintf("Installing recipe %s with prompts...", r.Name))
```

Comment:
`fmt.Sprintf` is unnecessary from within `log.Debugf` - you can use it the same way you would `Sprintf`.

Reply:
I was hoping for that, but it's not what I'm noticing when it runs with just the log.Debugf( ). In the output, I see the %s in the message, and the parameter added after the entire string.

Reply:
Ho my bad, I've mistaken log.DebugLn with log.Debugf.
Do I need to explicit do a line return if I use log.Debugf ?